skimage/color/colorconv.py

```diff
@@ -140,6 +140,9 @@ def _prepare_colorarray(arr, force_copy=False, *, channel_axis=-1):
     """
     arr = np.asanyarray(arr)
 
```

Comment:
We also might want to add here?

```suggestion
    _validate_channel_axis(channel_axis, arr.ndim)
```
The function is underused right now and would be useful in keeping `channel_axis` and `arr.ndim` matching.